opencodeblocks/graphics/edge.py

```diff
@@ -252,10 +252,8 @@ def deserialize(self, data: OrderedDict, hashmap: dict = None, restore_id=True):
         self.path_type = data["path_type"]
         try:
             self.source_socket = hashmap[data["source"]["socket"]]
-            self.source_socket.add_edge(self, is_destination=False)
 
             self.destination_socket = hashmap[data["destination"]["socket"]]
-            self.destination_socket.add_edge(self, is_destination=True)
             self.update_path()
```

Comment:
Won't this be a problem for copy & paste ?
Links need to be done with edges, I don't see where they are done now ...

Reply:
self.destination_socket = hashmap[data["destination"]["socket"]]
this uses a setter function which contains edge adding